server/report_builder/README.md

```diff
@@ -356,3 +356,33 @@ Reports can be built and tested directly from the 'open-msupply' repo, but devel
 Once changes are satisfactory, new reports can be moved directly into the OMS repo under a new version dir.
 
 Note that reports won't show up in OMS unless they are built into the generated json using the `build-standard-reports` command.
+
+## Localising JSON form fields
+
+Fields in UI schema can be translated using JSON forms inbuild translation and our translating function.
+
+Translations are invoked by adding by adding a identifying text key `T#` before a value in the json or ui schema.
+A translating function will step through the serialised json Value searching instances of this identifier. It will then strip it from the string, and translate the remaining string value.
+
+For example:
+
+```
+{
+"key": "T#label.value"
+}
+```
+
+becomes
+
+```
+{
+"key": "value"
+}
+```
+
+Where value is the translated value of 'label.value' in our `common.json` translation json in the front end.
+
+In translations for reports, the "label" key of the ui schema controls the text displayed in the front end.
```

Comment:
As per our covo, this was a little bit confusing, worth removing

Reply:
https://github.com/msupply-foundation/open-msupply/pull/5483/commits/ecbea1b9dcbdd5dc7d2b7fefac8e084109216f65#diff-da13630ce010bb5b9695835527e7e143a98cad61d3a39fdfc163117f8e0e1b28